cht-falcon/main.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import Callable
+
+import uvicorn
+from dotenv import load_dotenv
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from models import FalconBasedModel
+from routes import router as api_router
+
+load_dotenv()
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def create_start_app_handler(app: FastAPI) -> Callable[[], None]:
+    def start_app() -> None:
+        FalconBasedModel.get_model()
+
+    return start_app
+
+
+def get_application() -> FastAPI:
+    application = FastAPI(title="prem-chat", debug=True, version="0.0.1")
+    application.include_router(api_router, prefix="/api/v1")
```

Comment:
We removed /api/ from the roure defintion. You can keep only /v1

Reply:
oh cool! will update then